provisioning/containers/tigervnc/Dockerfile

```diff
@@ -0,0 +1,32 @@
+FROM ubuntu:20.04
+
+ARG USER=vncuser
+
+### ENV DEFAULTS
+ENV DISPLAY=:0 \
+    USER=${USER} \
+    DEBIAN_FRONTEND=noninteractive
+
+### Install required softwares & cleanup
+RUN apt-get update && \
+    apt-get install -y \
+        fluxbox \
+        tigervnc-standalone-server && \
+    apt-get --purge autoremove -y && \
+    apt-get autoclean -y
+
+EXPOSE 5900
+
+### Prepare usermode
+RUN useradd -ms /bin/bash $USER
```

Comment:
Here, I would explicitly configure a uid, since it is necessary to specify it in the k8s deployment.

Reply:
which uid has to be specified? 1000

Reply:
Really not sure about this. I would suggest 2000 to avoid matching the uid on the host, but I do not know if there is any "best practice"

@palexster what is your opinion?

Reply:
Do I have to specify uid and gid to match a similar pattern?
https://kubernetes.io/docs/tasks/configure-pod-container/security-context/

Reply:
Exactly, the idea is to configure the uid you choose in the security-context of the pod

Reply:
So, I checked in the only container that has the user already set, that is the nginx one. The uid there is 101. 
I was wondering for praticity if we can set 101 as the default uid in all the other containers at this point...

Reply:
I would avoid assigning a uid < 1000 to a "human" user as it is typically used for services, even though this distinction probably is not that important in k8s

Reply:
Ok but at this point I have to change the uid for the nginx user (in the nginx container) I guess?

Reply:
As it is possible to [specify a different securityContext for each container](https://kubernetes.io/docs/tasks/configure-pod-container/security-context/#set-the-security-context-for-a-container) in a pod, that should be not necessary

Reply:
I added a new user on the nginx container and it seems to work, I'm pushing the changes now